utils/systemd-redis_multiple_servers@.service

```diff
@@ -1,26 +1,26 @@
-# example systemd template service unit file for multiple redis-servers
+# example systemd template service unit file for multiple placeholderkv-server
```

Comment:
As in every other file, I think we shouldn't use "placeholderkv" more than once in each script for defining a variable at the top of the file.

In comments, we can just say server instance and similar words.